# <font color="#008080"> ***Diplomado: Ciencia de datos*** </font>
## <font color="#008000"> **Diplomado: Módulo 5** </font>
### <font color="#000080"> **Diplomado: Práctica 2** </font>
#### <font color="#800080"> **Diplomado: Karla Ivonne Flores Cisneros** </font>

# **Objetivo: Aprender a cargar diferentes tipos de archivos usando la sintaxis/instrucción apropiada**











In [ ]:
import pandas as pd

# **A) Leer Archivo_1.CSV**

a.2 ) Archivo 1 como dataframe  contiene 5 columnas


a.3 ) Y 1000 filas o registros, es decir 5000 objetos.










In [ ]:
archivo_1=pd.read_csv('datosP2/archivo_1.csv')

In [ ]:
archivo_1.head(2)

x   o   r      y      t
0  88  12  96  80.55  aeeaa
1  17  21  76   8.32  oieia

In [ ]:
archivo_1.tail(2)

x   o   r      y      t
998  37  68  64  43.59  uieiu
999  11  29  89  15.12  oueaa

In [ ]:
len(archivo_1.index)

1000

In [ ]:
len(archivo_1.columns)

5

In [ ]:
archivo_1.shape

(1000, 5)

In [ ]:
1000*5

5000

# **B) Leer Archivo_2.CSV**

b.2 ) Archivo 2 como dataframe contiene 5 columnas

b.3 ) Y 1000 filas o registros, es decir 5000 objetos.










In [ ]:
archivo_2=pd.read_csv('datosP2/archivo_2.csv')

In [ ]:
archivo_2.head(2)

k   j   c      n      m
0  74  15  87  38.31  uoaea
1  21  15  62  79.42  oaiua

In [ ]:
archivo_2.tail(2)

k   j   c      n      m
998  55  98  95   8.22  ueoie
999  84  35   1  10.70  uieoa

In [ ]:
len(archivo_2.index)

1000

In [ ]:
len(archivo_2.columns)

5

In [ ]:
archivo_2.shape

(1000, 5)

In [ ]:
1000*5

5000

# **C) Leer Archivo_3.CSV**

c.2 ) Archivo 3 como dataframe contiene 5 columnas

c.3 ) Y 1000 filas o registros, es decir 5000 objetos.










In [ ]:
archivo_3=pd.read_csv('datosP2/archivo_3.csv')

In [ ]:
archivo_3.head(2)

z   u   o      y      p
0  32  88  71  38.28  uuauo
1   6  46   1  19.05  ioaei

In [ ]:
archivo_3.tail(2)

z   u   o      y      p
998  85   7  96  42.62  oueoa
999  34  41  86   6.80  uiaii

In [ ]:
len(archivo_3.index)

1000

In [ ]:
len(archivo_3.columns)

5

In [ ]:
archivo_3.shape

(1000, 5)

In [ ]:
1000*5

5000

# **D) Leer Archivo_4.TXT**

d.2 ) Archivo 4 como dataframe contiene 1 columna


d.3 ) Y 1250 filas o registros, es decir 1250 objetos.










In [24]:
archivo_4=pd.read_csv('datosP2/archivo_4.txt', sep='\t')

In [25]:
archivo_4.head()

b,z,v,q,w
0  10,58,100,17.09,eieai
1    52,47,52,6.52,auuoe
2   63,20,44,54.09,oeoou
3   71,20,92,70.01,eiiai
4   97,29,79,81.61,aieee

In [33]:
archivo_4.tail()

b,z,v,q,w
1245    75,87,13,6.2,aeoii
1246  74,78,10,64.48,oaaae
1247  61,82,30,10.94,aoouo
1248  73,11,56,42.53,oaiao
1249  11,92,29,39.68,oaioa

In [27]:
len(archivo_4.index)

1250

In [28]:
len(archivo_4.columns)

1

In [29]:
archivo_4.shape

(1250, 1)

In [30]:
1250*1

1250

# **E) Leer Archivo_5.TXT**

e.2 ) Archivo 5 como dataframe contiene 1 columna

e.3 ) Y 1525 filas o registros, es decir 1525 objetos.

In [31]:
archivo_5=pd.read_csv('datosP2/archivo_5.txt', sep='\t')

In [32]:
archivo_5.head()

o,d,l,e,k
0   54,49,67,62.5,euoee
1   57,79,30,73.0,ouaai
2  57,61,33,82.48,aueee
3  29,89,39,46.22,ooaia
4  21,97,56,85.06,aaaie

In [34]:
archivo_5.tail()

o,d,l,e,k
1520  15,97,62,54.47,euuau
1521  90,43,93,42.28,ueooi
1522  84,23,64,97.15,iooua
1523   81,84,90,92.2,uaiiu
1524   25,6,16,96.39,aoeeu

In [35]:
len(archivo_5.index)

1525

In [36]:
len(archivo_5.columns)

1

In [37]:
archivo_5.shape

(1525, 1)

In [38]:
1525*1

1525

## **1) Unir dataframes en un solo archivo**

In [39]:
datosP2_union=pd.concat([archivo_1,archivo_2,archivo_3,archivo_4,archivo_5])

In [43]:
datosP2_union.head()

x     o     r      y      t   k   j   c   n    m   z   u    p b,z,v,q,w  \
0  88.0  12.0  96.0  80.55  aeeaa NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
1  17.0  21.0  76.0   8.32  oieia NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
2  39.0  90.0  25.0  78.91  eiooe NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
3  67.0  71.0  87.0  55.07  aeooe NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
4  10.0  73.0  90.0  35.84  eooeu NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   

  o,d,l,e,k  
0       NaN  
1       NaN  
2       NaN  
3       NaN  
4       NaN

In [48]:
vvv.tail()

x   o   r   y    t   k   j   c   n    m   z   u    p b,z,v,q,w  \
1520 NaN NaN NaN NaN  NaN NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
1521 NaN NaN NaN NaN  NaN NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
1522 NaN NaN NaN NaN  NaN NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
1523 NaN NaN NaN NaN  NaN NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   
1524 NaN NaN NaN NaN  NaN NaN NaN NaN NaN  NaN NaN NaN  NaN       NaN   

                 o,d,l,e,k  
1520  15,97,62,54.47,euuau  
1521  90,43,93,42.28,ueooi  
1522  84,23,64,97.15,iooua  
1523   81,84,90,92.2,uaiiu  
1524   25,6,16,96.39,aoeeu

In [50]:
datosP2_union

x     o     r      y      t   k   j   c   n    m   z   u    p  \
0     88.0  12.0  96.0  80.55  aeeaa NaN NaN NaN NaN  NaN NaN NaN  NaN   
1     17.0  21.0  76.0   8.32  oieia NaN NaN NaN NaN  NaN NaN NaN  NaN   
2     39.0  90.0  25.0  78.91  eiooe NaN NaN NaN NaN  NaN NaN NaN  NaN   
3     67.0  71.0  87.0  55.07  aeooe NaN NaN NaN NaN  NaN NaN NaN  NaN   
4     10.0  73.0  90.0  35.84  eooeu NaN NaN NaN NaN  NaN NaN NaN  NaN   
...    ...   ...   ...    ...    ...  ..  ..  ..  ..  ...  ..  ..  ...   
1520   NaN   NaN   NaN    NaN    NaN NaN NaN NaN NaN  NaN NaN NaN  NaN   
1521   NaN   NaN   NaN    NaN    NaN NaN NaN NaN NaN  NaN NaN NaN  NaN   
1522   NaN   NaN   NaN    NaN    NaN NaN NaN NaN NaN  NaN NaN NaN  NaN   
1523   NaN   NaN   NaN    NaN    NaN NaN NaN NaN NaN  NaN NaN NaN  NaN   
1524   NaN   NaN   NaN    NaN    NaN NaN NaN NaN NaN  NaN NaN NaN  NaN   

     b,z,v,q,w             o,d,l,e,k  
0          NaN                   NaN  
1          NaN                   NaN  
2          NaN                   NaN  
3          NaN                   NaN  
4          NaN                   NaN  
...        ...                   ...  
1520       NaN  15,97,62,54.47,euuau  
1521       NaN  90,43,93,42.28,ueooi  
1522       NaN  84,23,64,97.15,iooua  
1523       NaN   81,84,90,92.2,uaiiu  
1524       NaN   25,6,16,96.39,aoeeu  

[5775 rows x 15 columns]

In [46]:
datosP2_union.shape

(5775, 15)

In [51]:
datosP2_union.to_csv('datosP2_union.csv',index=False)

## **4) ¿Cuántos registros tiene el Dataframe resultante?**

el Dataframe resultante datosP2 tiene 5775 filas y 15 columnas, solo que solo se imprimen hasta la 1524.

